Workbooks/Azure Sentinel/Overview/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
+    "name":"Azure Sentinel Overview",
+    "author": "Microsoft",
+    "galleries": [{ "type": "Sentinel-insights", "resourceType": "Azure Monitor", "order": 100 }]
```

Comment:
for the gallery type, change this to `sentinel` since we already have sentinel workbook types